### PR TITLE
add getStatistics

### PR DESCRIPTION
**Pull request by DarinaSednevets**
res 
{
    "status": "success",
    "code": 200,
    "data": {
        "year": "2019",
        "month": "2",
        "transactions": [
            {
                "_id": "620cf2178f046ddb71fc72c9",
                "year": 2019,
                "category": "car",
                "sum": 100,
                "month": 2
            }
        ],
        "uniqueCategories": [
            "car"
        ],
        "categoriesSumm": {
            "car": 100
        }
    }

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ require("dotenv").config();
 // const usersRouter = require("./routes/api/users");
 const transactionsRouter = require("./routes/api/transactions");
 const categoriesRouter = require("./routes/api/categories");
+const statisticsRouter = require("./routes/api/statistics");
```
Comment (Prykhodko-Pasha) [inline app.js]: we don't have such route. please watch swagger file

File: app.js [hunk 2]
```diff
@@ -19,6 +20,7 @@ app.use(express.json());
 // app.use("/api/users", usersRouter);
 app.use("/api/transactions", transactionsRouter);
 app.use("/api/categories", categoriesRouter);
+app.use("/api/statistics", statisticsRouter);
```
Comment (Prykhodko-Pasha) [inline app.js]: the same is here

File: controllers/statistics/getStatistics.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    return res.json({
+        status: "success",
+        code: 200,
+        data: {
+            year,
+            month,
+            transactions: filteredResultByMonth,
+            uniqueCategories,
+            categoriesSumm,
+        }
```
Comment (Prykhodko-Pasha) [inline controllers/statistics/getStatistics.js]: res is not corresponding to our swagger file

File: controllers/statistics/index.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const getStatistics = require("./getStatistics");
+
+module.exports = {
+    getStatistics
+};
```
Comment (Prykhodko-Pasha) [inline controllers/statistics/index.js]: please move this controller to transactions

File: routes/api/statistics.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+const express = require("express")
+const router = express.Router()
+const { getStatistics } = require("../../controllers/statistics");
+
+router.get("/", getStatistics);
+
+module.exports = router;
+
```
Comment (Prykhodko-Pasha) [inline routes/api/statistics.js]: we don't have such route. please watch swagger file

**Review by Prykhodko-Pasha — APPROVED** [review]

**Review by SovaV — APPROVED** [review]

**Pull request merged**